modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/servlet/LayoutStructureCommonStylesCSSServlet.java

```diff
@@ -316,6 +316,10 @@ private JSONObject _getFrontendTokensJSONObject(
 				value = valueJSONObject.getString("value");
 			}
 
+			if (value == null) {
+				value = frontendToken.getDefaultValue();
+			}
+
```

Comment:
Hi @dantewang, I saw `JSONObject.getString()` method has a default value -> `""`, so it won't be null.

Reply:
@JanisZhang Hmm, OK, if so, we don't need this one.